.gitignore

```diff
@@ -1,6 +1,7 @@
+.DS_Store
```

Comment:
As this is OS-specific, rather than ignoring it in every repository you work with, it should go in a global ignore file on your machine. For example, I have `~/.gitignore` on my machine with the following contents:

```
.DS_Store
```

`core.excludesfile` is configured to point to it:

```
$ git config --global core.excludesfile
/Users/awilkinson/.gitignore
```

You can set it with the following:

```
$ git config --global core.excludesfile ~/.gitignore
```

Reply:
Done. Thanks.